languages/java/exercises/concept/numbers/.docs/after.md

```diff
@@ -0,0 +1,85 @@
+One of the key aspects of working with numbers in Java is the distinction between integers (numbers with no digits after the decimal separator) and floating-point numbers (numbers with zero or more digits after the decimal separator).
+
+Java has other [datatypes][numeric-datatypes] apart from `int` and `double`
+
+```Java
```

Comment:
the expected label for block syntax highlight is lower-case, so `java` please fix it in all markdown files